### PR TITLE
BITMAKER-1839: Improve swagger documentation

### PR DESCRIPTION
**Pull request by rodp63**
Ticket: https://tasks.bitmaker.dev/issues/1839

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -3,15 +3,25 @@
...
 
 Install Bundler and Jekyll (if you haven't)
-```
+```bash
 $ gem install bundler jekyll
 ```
-Install the gems if it's the first time you build the project
+Install Redoc CLI (if you haven't)
+```bash
+$ yarn global add redoc-cli
```
Comment (mgonnav) [inline docs/README.md:11]: Any requirement for the node version?

**Review by mgonnav — COMMENTED** [review]
It looks excellent. Is there any way we can add a button back to the main docs?

**Review by mgonnav — CHANGES_REQUESTED** [review]
.

File: docs/README.md (hunk 1 above)
Comment (rodp63) [reply docs/README.md:11]: Not sure 🤔, I'm currently using v17.8.0

**Review by mgonnav — APPROVED** [review]
Nicely done! 👍🏻

**Pull request merged**